### DISCUSSION TITLE
Stand Up Monday June 5th

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#87] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- finished 2 deployment activities with @CristianCazares 
- Made some progress on Architecture doc
- Tried to implement singleton pattern for having one db instance on backend

**What are your priorities for the next 24 hours?**:
- Keep working on architecture doc
- Tried to checked what happened with the singleton

**Blockers**:
- lack of will

**Shoutouts**:
- @Jacobgtz024 for going to raw alejandro's concert

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Finished graph of most attended certifications
- Worked on architecture document

**What are your priorities for the next 24 hours?**:
- Work on more graphs

**Blockers**:
- None

**Shoutouts**:
- @CarlosCRG19 he a good fella

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Refactor code
- Cassandra 2?

**What are your priorities for the next 24 hours?**:
- Reach out @CarlosCRG19 to communicate the rest of the graphs

**Blockers**:
- None

**Shoutouts**:
- @BrandonMagana and @CristianCazares for the friday grind

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on Dashboard Tasks
- Create the layout for Searchable Radar graph

**What are your priorities for the next 24 hours?**:
- Work on HTTPClient refactor
- Meet with @CarlosCRG19 and @Jacobgtz024 to set a workflow to get every Jacob graph into the front ASAP

**Blockers**:
- Too much documents

**Shoutouts**:
- @BrandonMagana for getting his Driver's License last week just like we always talked about cause you were so excited for me to finally drive up to your house
